src/views/virtualmachines/details/tabs/configuration/ssh/components/SSHTabSSHAccess.tsx

```diff
@@ -0,0 +1,30 @@
+import React, { useState } from 'react';
+
+import LinuxLabel from '@kubevirt-utils/components/Labels/LinuxLabel';
+import SSHAccess from '@kubevirt-utils/components/SSHAccess/SSHAccess';
+import useSSHService from '@kubevirt-utils/components/SSHAccess/useSSHService';
+import { useKubevirtTranslation } from '@kubevirt-utils/hooks/useKubevirtTranslation';
+import { ExpandableSection } from '@patternfly/react-core';
+
+const SSHTabSSHAccess = ({ vm, vmi }) => {
+  const { t } = useKubevirtTranslation();
+  const [sshService, sshServiceLoaded] = useSSHService(vm);
+  const [isExpanded, setIsExpanded] = useState<boolean>();
+
+  return (
+    <ExpandableSection
+      toggleContent={
+        <>
+          {t('SSH Access')} <LinuxLabel />
```

Comment:
As mentioned above, 'Access' should be all lowercase here.

Reply:
changed